adl/config.adl

```diff
@@ -32,7 +35,7 @@ struct ToolConfig {
     StringKeyMap<StaticConfigName, JsonSource> configSources = {};
 
     /// Dynamic config sources
-    StringKeyMap<DynamicConfigName, DynamicJsonSource> dynamicConfigSources = {};
+    DynamicConfigNameJSrcMap dynamicConfigSources = {};
```

Comment:
Did you consider merging configSource and dynamicConfigSources into a single map? What does it mean if I have 2 keys the same in both maps?

Reply:
IIRC there was discussion and a decision to keep `ToolConfig` `configSources` and `dynamicConfigSources` separate explicitly.
This landed in https://github.com/helix-collective/hx-deploy-tool/pull/79/files

See also https://github.com/helix-collective/hx-deploy-tool/issues/31

Reply:
OK.